evergreen/evergreen.yml

```diff
@@ -217,6 +217,16 @@ functions:
           ${PREPARE_SHELL}
           AUTH=${AUTH} SSL=${SSL} MONGODB_URI="${MONGODB_URI}" TOPOLOGY=${TOPOLOGY} OS=${OS} evergreen/run-tests.sh
 
+  run-atlas-connectivity-tests:
```

Comment:
Added three new things to evergreen.yml:

1. A new function called run-atlas-connectivity-tests
2. A new task called atlas-connectivity-tests
3. A new build variant called atlas-connectivity-tests